### ISSUE TITLE
Future feature?  Enchance one of the "Charts" to spawn CardViews?

### ISSUE THREAD

**Issue by fxtmtrsine** [cdhorn/CardView#153]
I really like how you use **Child object indicators**  and **Status indicators** and imagine that enhancing one of the existing "Charts" to spawn CardViews would be popular (at least with me)? 

I'm getting a real "[Kith and Kin](https://en.wikipedia.org/wiki/Kith_and_Kin_Pro)" feel out of Cardview are you a past user of the program, as I could really see the  main K&K navigation screen making a great chart?

<img src="https://upload.wikimedia.org/wikipedia/commons/0/02/Kith_and_Kin_Pro_V3.3.0_screenshot.png" width=50% height=50%>

**Comment by cdhorn** [comment]
Thanks, the idea is to try to convey as much about the state of the object as possible and then trying to find a good way to navigate or access the information.

I'm going to close this as out of scope for now as changes need to be made to Gramps core for things like that. Nick has indicated we may fold this in at some point, but some groundwork needs to be done first to make things work smoother.

Ideally someday I agree it would be nice to have a tree or fan icon perhaps and be able to pivot to the pedigree or fan chart and from there pivot back.

No have never seen Kith and Kin, will need to look at it when I have some spare cycles.
